sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -33,6 +33,11 @@
 #include "debug.h"
 
 /**
+ * @brief Helper macro for LL_SEARCH to compare TCBs
+ */
+#define TCB_EQUAL(a,b)      ((a) == (b))
+
+/**
```

Comment:
@smlng in one @brummer-simon was right, that it should return `0` in the *equal* case (sorry saw your reply only after I posted my comment)